Nozomi/nozomi-vantage/ingest/parser.yml

```diff
@@ -0,0 +1,149 @@
+name: nozomi-vantage
+pipeline:
+  - name: json_event
+    external:
+      name: json.parse-json
+
+  - name: parsed_date
+    external:
+      name: date.parse
+      properties:
+        input_field: "{{json_event.message.time}}"
+        output_field: datetime
+
+  - name: parsed_update_time
+    external:
+      name: date.parse
+      properties:
+        input_field: "{{json_event.message.update_time}}"
+        output_field: datetime
+    filter: "{{json_event.message.get('update_time') != None}}"
+
+  - name: set_ecs_fields
+
+  - name: set_alert_fields
+    filter: "{{json_event.message.event_type == 'alerts'}}"
+
+  - name: set_asset_fields
+    filter: "{{json_event.message.event_type == 'assets'}}"
+
+  - name: set_vulnerability_fields
+    filter: "{{json_event.message.event_type == 'vulnerabilities'}}"
+
+  - name: set_wireless_fields
+    filter: "{{json_event.message.event_type == 'wireless_networks'}}"
+
+stages:
+  set_ecs_fields:
+    actions:
+      - set:
+          "@timestamp": "{{parsed_date.datetime}}"
+          event.dataset: "{{json_event.message.event_type}}"
+          event.type: ["info"]
+
+      - translate:
+        dictionary:
+          "alerts": "alert"
+          "assets": "asset"
+          "vulnerabilities": ["vulnerability"]
```

Comment:
I think that this value is unexpected here (and it's not a correct one according to the ECS specifciation)
```suggestion
```